src/components/LiquidityWidget/LiquidityWidget.tsx

```diff
@@ -486,7 +445,7 @@ const LiquidiyWidget: React.FC<ILiquidityWidgetProps> = ({ onCancel }) => {
                   <Flex sx={{ justifyContent: 'space-between', columnGap: '15px' }}>
                     {approvalA !== ApprovalState.APPROVED && (
                       <Button
-                        csx={styles.button}
+                        sx={styles.button}
```

Comment:
no pragma in this file too, can you check others too

Reply:
Done